### ISSUE TITLE
[ContentAssistance] Global variables are not visible within nested loops

### ISSUE THREAD

**Issue by dilinisg** [ballerinalang/composer#2359]
Suggestions are not loaded for lname and age inside the while loop

```
package routingServices.samples;

import ballerina.net.http;
import ballerina.lang.jsons;
import ballerina.lang.messages;

string lname;
int age;

@http:config {basePath:"/cbr"}
service<http> contentBasedRouting {
    
    @http:POST{}
    @http:Path {value:"/"}
    resource cbrResource (message m) {
        http:ClientConnector nasdaqEP = create http:ClientConnector("http://localhost:9090/nasdaqStocks");
        http:ClientConnector nyseEP = create http:ClientConnector("http://localhost:9090/nyseStocks");
        string nyseString = "nyse";
        
        json jsonMsg = messages:getJsonPayload(m);
        string nameString = jsons:getString(jsonMsg, "$.name");
        message response = {};
        
        if (nameString == nyseString) {
            response = http:ClientConnector.post(nyseEP, "/stocks", m); 
            
            string name;  
            while (true) {
                  // No content assistance for lname, age
            }        
        }
        else {
            response = http:ClientConnector.post(nasdaqEP, "/stocks", m);
        
        }
        reply response;
        
    }
    
}

```

**Comment by dilinisg** [comment]
This is not happening because they are nested loops, but because the while is not recognizing its scope

**Comment by nadeeshaan** [comment]
Fixed in v0.90. Hence closing the issue
